### ISSUE TITLE
[fvt]2.12.1 xcatprobe image need  modify usage

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1401]
env:ubuntu
build:lastest build.
Version 2.12.1 (git commit 5431a2ab9ccd42f5f76540ec0868429dd6199056, built Sun Jun 26 06:47:12 EDT 2016)
How to reproduce:
```
                  supported are redhat, sles and ubuntu. 
root@c910f03c05k05:/opt/xcat/bin# xcatprobe  -image -h
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 64.
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 173.
Unsupported attribute: -image
Usage:
xcatprobe -h
xcatprobe -l
xcatprobe [-n] [-V] <subcommand>  <attrbute_to_subcommand>

Options:
    -h : get usage information of xcatprobe 
    -l : list all valid sub commands
    -V : print verbose information of xcatprobe
    -n : print output without colors 
root@c910f03c05k05:/opt/xcat/bin# lsdef -v

```
---------------------> I could not get the point the image only for diskless image so please modify the usage

**Comment by junxiawang** [comment]
root@c910f03c05k05:/opt/xcat/bin# xcatprobe image -l 
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 64.
Smartmatch is experimental at /opt/xcat/bin/xcatprobe line 173.
Unknown option: l
[FAIL] Invalid parameter for image
       Usage:
    image -h
    image -t
    image {-c|-d} [-V]

Description:
    Use this command to check if compute nodes have the same images installed as defines in xCAT DB. 
    Use this command to check if all compute nodes have the same identical installed. 

Options:
    -h : Get usage information of image
    -t : To verify if image can work, reserve option for probe framework
    -d : To verify compute nodes have the same images installed as defines in xCAT DB. 
    -c : To verify compute nodes have the identical images installed.
    -V : To print additional debug information.

root@c910f03c05k05:/opt/xcat/bin#

**Comment by gurevichmark** [comment]
I am already making changes to `image` probe based on Xiaopeng's suggestions. Will fix this under the same pull request.

**Comment by gurevichmark** [comment]
Fixed under #1406
